rtest.py

```diff
@@ -366,6 +385,35 @@ def remove_test(args):
           .format(name))
 
 
+def clean_up(args):
+    '''
+    Delete temporary testing files.
+
+    Temporary testing files include:
+
+    1) customized grammars
+    2) current profiles
+    3) logs
+    '''
+    for name, _, _, _, _, prof, _ in _discover(args):
+        deleted = False
+        grm = GRAMMARS_DIR / name
+        if grm.exists():
+            shutil.rmtree(grm)
+            deleted = True
+        if prof and prof.exists():
+            shutil.rmtree(prof)
+            deleted = True
+        # a test name may be a substring of another test name, so use a
+        # glob pattern that looks like a date to mitigate this
+        date_glob = '[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]'
+        for log in LOGS_DIR.glob(f'{name}-{date_glob}*'):
+            log.unlink()
+            deleted = True
+        if deleted:
+            print(f'cleaned files for {name}')
```

Comment:
I am on the fence about printing out each deleted filename; on the one hand, this is transparent (good); on the other, with thousands of files, this will be a bit annoying. After all, you ran the command in order to get rid of them quickly. To print out several thousand filenames may take more than a minute, and at that point might as well delete them manually?

Reply:
It only prints the test name for which files were deleted, not for every file. Manually deleting (typing or globbing every path, or building a `find` query, making sure not to delete the `README.md` files, etc.) them takes much longer than the time to print the test names. Personally I find it a bit reassuring to see what happened, but if it's really annoying we can hide it unless `--verbose` is used.

Reply:
> It only prints the test name for which files were deleted, not for every file.

Yeah, that's thousands, if you haven't cleared your logs for a while.



> Manually deleting (typing or globbing every path, or building a `find` query, making sure not to delete the `README.md` files, etc.) them takes much longer than the time to print the test names.

I don't think so, actually! I usually just restore the READMEs by stashing local changes. Takes a few seconds, but of course is still annoying to have to do by hand.

> Personally I find it a bit reassuring to see what happened, but if it's really annoying we can hide it unless `--verbose` is used.

This would be my preference fwiw :).

Reply:
> Yeah, that's thousands, if you haven't cleared your logs for a while.

No. We have ~475 tests that run (the skipped tests don't produce logs). Like I said, only the test name is printed, not each file.

Reply:
If the number of lines printed is the number of tests maximum every time, then it is less of a problem, though I still think it makes the process slower (but still automatic) than removing everything manually via Finder (particularly removing just the logs, which is all I usually need).

Reply:
Thank you! Love the new --clean option :).

Reply:
Oh! Something just occurred to me. If you'd rather have the default --clean option be verbose (say it speaks to you conceptually, you believe such a command should by default be printing out filenames), I'd be completely happy with an additional option like --clean --suppressed or whatever. So long as I can quickly delete everything and not wait, that'll work.

Reply:
I just think that destructive actions like this shouldn't always work silently. The current solution is fine.